test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml

```diff
@@ -299,3 +299,42 @@ tests:
     alert_rule_test:
       - eval_time: 10m
         alertname: ChainsControllerDeadlock
+
+
+  # ----- PAC Controller Potential Deadlock ----
```

Comment:
before we talk about this stuff voice to voice give your prior comment about being uncertain about this stuff, I'll check out your branch and try to run this locally, see how close you got .... if only minor changes are needed, I'll make suggestions in a separate review.